squish.user.js

```diff
@@ -0,0 +1,291 @@
+
+  let column_headers = document.querySelectorAll('table.client-table thead th');
+  let name_header_index = 1;
+  let id_index = 2;
+  let org_header_index = 3;
+  let lang_header_index = 4;
+  let unemployment_index = 5;
+  let updated_index = 6;
+  let waiting_index = 7;
+  let created_index = 8;
+  for ( let header_i = 0; header_i < column_headers.length; header_i++ ) {
+    let text = column_headers[ header_i ].innerText;
+    if ( text === 'Name' ) { name_header_index = header_i; }
+    if ( text === 'Client ID' ) { id_index = header_i; }
+    if ( text === 'Organization' ) { org_header_index = header_i; }
+    if ( text === 'Language' ) { lang_header_index = header_i; }
+    if ( text === 'UI' ) { unemployment_index = header_i; }
+    // Don't change wording of headings right now. The current headings
+    //  are in the documentation. Just decrease width
+    if ( text === 'Updated At' ) {
+      updated_index = header_i;
+      column_headers[ header_i ].querySelector('span').innerHTML = 'Updated<br>at';
+    }
+    if ( text === 'Waiting on response' ) {
+      waiting_index = header_i;
+      column_headers[ header_i ].querySelector('span').innerHTML = 'Waiting on<br>response';
+    }
+    if ( text === 'Created at' ) { created_index = header_i; }
+  }
+
+  let new_info_data = [];
+
+  let info_rows = document.querySelectorAll('table.client-table tbody tr');
+  for ( let row of info_rows ) {
+
+    let cols = row.querySelectorAll(':scope > *');
+
+    let client_id = cols[ id_index ].innerText;
+    let id_container = document.createElement('span');
+    id_container.className = 'bookmarklet name_id_info';
+    id_container.innerHTML = `(${ client_id })`
+
+    let org_name = cols[ org_header_index ].innerText;
+
+    let name_subtitle_container = document.createElement('div');
+    name_subtitle_container.innerHTML = `<span class="bookmarklet_org_subtitle">${ org_name }</span>`;
+    
+    let name_cell = cols[ name_header_index ];
+    let name_node = name_cell.querySelectorAll( 'a' )[0];
+    name_node.parentNode.insertBefore( id_container, name_node.nextSibling );
+    name_cell.appendChild( name_subtitle_container );
+
+    // Change dates to days, but keep date data around in a tooltip
+    let updated_days = '-';
+    let waiting_days = '-';
+    let created_days = '-';
+
+    let updated_text = cols[ updated_index ].innerText;
+    let waiting_text = cols[ waiting_index ].innerText;
+    let created_text = cols[ created_index ].innerText;
+
+    if ( updated_text ) {
+      let date = new Date( updated_text + ` ${this_year}` );
+      updated_days = Math.round(( today - date) / one_day_ms);
+    }
+    if ( waiting_text ) {
+      let date = new Date( waiting_text + ` ${this_year}` );
+      waiting_days = Math.round(( today - date) / one_day_ms);
+    }
+    if ( created_text ) {
+      let date = new Date( created_text + ` ${this_year}` );
+      created_days = Math.round(( today - date) / one_day_ms);
+    }
```

Comment:
Minor, and optional: these 3 calls would probably be clearer (and easier to maintain) as a function, called 3 times.

Reply:
Created #31